frontend/tasks/pages.js

```diff
@@ -95,7 +95,8 @@ function buildExperimentPage() {
 
 function convertToCompiledPage() {
   return through.obj(function compiledConvert(file, encoding, callback) {
-    const filename = file.path.split('/').pop();
+    //const filename = file.path.split('/').pop();
```

Comment:
No need to comment this out, if there are problems we can always check the commit log to find the previous version.